src/navigation/dispatch/useAllTasks.js

```diff
@@ -1,21 +1,22 @@
 import { useDispatch } from "react-redux";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 
 import {
   loadTaskListsSuccess,
   loadUnassignedTasksSuccess,
   loadUsersSuccess,
 } from "../../redux/Dispatch/actions";
-import { useGetCourierUsersQuery } from '../../redux/api/slice';
 import {
+  useGetCourierUsersQuery,
   useGetTaskListsQuery,
   useGetUnassignedTasksQuery,
 } from "../../redux/api/slice";
 
 
-export function useLoadUnassignedTasksInfo(date) {
+export function useAllTasks(date) {
 
   const dispatch = useDispatch();
+  const [isFirstLoad, setIsFirstLoad] = useState(true)
```

Comment:
I don't understand why we need to manage it manually

I think we can use `isFirstLoad = isLoadingTaskLists || isLoadingUnassignedTasks || isLoadingCourierUsers`

because 

> isLoading - When true, indicates that the query is currently loading for the first time, and has no data yet. This will be true for the first request fired off, but not for subsequent requests

from https://redux-toolkit.js.org/rtk-query/usage/queries#frequently-used-query-hook-return-values